### PR TITLE
Update firebase token script

### PR DESCRIPTION
**Pull request by ogonommo**
## Какво променя този PR?

Fix за генериране на id токени от Firebase.

## Детайли

Импортите са конвертирани в CommonJS импорти, скриптът се изпълнява директно през node, където не хваща конфигурационни файлове и default-ва до CommonJS.

### CODE REVIEW

File: scripts/get-token.js [hunk 1]
```diff
@@ -1,23 +1,59 @@
...
+//     console.error(error)
+//     process.exit(1)
+//   }
+// }
+// getIdtoken()
+
+const fs = require('fs')
+const admin = require('firebase-admin')
+const dotenv = require('dotenv')
+const serviceAccount = require('../firebase.json')
```
Comment (hkdobrev) [inline scripts/get-token.js:5]: Location of this file should be located via `process.env.GOOGLE_APPLICATION_CREDENTIALS`.

File: scripts/get-token.js [hunk 2]
```diff
@@ -1,23 +1,33 @@
-import fs from 'fs'
-import dotenv from 'dotenv'
-import FirebaseTokenGetter from 'firebase-idtoken-getter'
```
Comment (hkdobrev) [inline scripts/get-token.js:3]: Could we remove this dependency from `package.json`? It's not really being kept up to date.

**Review by hkdobrev — CHANGES_REQUESTED** [review]

File: scripts/get-token.js [hunk 3]
```diff
@@ -1,23 +1,33 @@
...
-  try {
-    const token = await FirebaseTokenGetterObject.createIdTokenBycustomToken()
-    console.log(token)
-  } catch (error) {
-    console.error(error)
+admin
+  .auth()
+  .createCustomToken(envConfig.FIREBASE_UID, {})
+  .then((token) => {
+    return fetch(
```
Comment (hkdobrev) [inline scripts/get-token.js:17]: `fetch` is not defined for me as I'm using node 16 as on prod server. We could eventually upgrade to node 18+, but for now could you use a polyfill like `node-fetch`?

**Review by hkdobrev — CHANGES_REQUESTED** [review]

File: scripts/get-token.js (hunk 3 above)
Comment (hkdobrev) [reply scripts/get-token.js:17]: пробвах го на node 18 и работи супер, но все пак ми се иска да работи и на node 16, засега

Comment (ogonommo) [reply scripts/get-token.js:19]: Installed version 2.4 of `node-fetch` as it's the most current one supporting CommonJS modules, we might want to upgrade the version if the code is refactored to work with ES modules.

File: package.json [hunk 4]
```diff
@@ -30,7 +30,7 @@
     "migration:revert": "typeorm-ts-node-esm -d ./src/config/typeorm.datasource.ts migration:revert",
     "seed:run": "typeorm-ts-node-esm -d ./src/config/typeorm-seeds.datasource.ts migration:run",
     "seed:revert": "typeorm-ts-node-esm -d ./src/config/typeorm-seeds.datasource.ts migration:revert",
-    "firebase:token": "node ./scripts/get-token.js",
+    "firebase:token": "node --experimental-loader ./scripts/get-token.js",
```
Comment (hkdobrev) [inline package.json:33]:
```suggestion
    "firebase:token": "node ./scripts/get-token.js",
```
I think this works without this option on both Node 16 and Node 18.
It produces extra output which is not easily silenced and it prevents me from using it programmatically with my httpie flow with:
```shell
export JWT_AUTH_TOKEN=$(npm run --silent firebase:token)`
```

**Review by hkdobrev — CHANGES_REQUESTED** [review]

**Review by hkdobrev — APPROVED** [review]
🙌 👏

**Pull request merged**